cypress/integration/touch.spec.ts

```diff
@@ -71,19 +71,4 @@ describe("cy.realTouch", () => {
       })
       .realTouch({ radiusX: 5, radiusY: 7 });
   });
-
-  it("touches using provided 0 for one of the axis", { retries: 4 }, (done) => {
```

Comment:
@dmtrKovalenko any particular reason why this test was removed? it was added as part of one of my PRs since this was not working prior to that PR

Reply:
Nope, I am trying to figure out the sigsev fialures that happens on CI and does not reproduce locally. Actually I have no idea why it happens, it can happen in a lot of situations which is not captured. I think this may be related to `async` function which then handled by bluebird, need to try get rid of `await` and use custom commands. 

CI is still flaky even in the `main` which makes me mad (need to rerun from fialure like 5 times).